provision/internal/operator/terraform/operator.go

```diff
@@ -36,7 +36,11 @@ func (t *Terraform) Create(p types.ProviderType, cfg map[string]interface{}) (*t
 	// silence stdErr during terraform execution, plugins send debug and trace entries there
 	if !t.ops.Verbose {
 		stderr := os.Stderr
-		os.Stderr, _ = os.Open(os.DevNull)
+		var err error = nil
```

Comment:
```suggestion
		var err error
```

Zero value of an error is already nil